R/tlf_ae_specific.R

```diff
@@ -53,10 +54,10 @@ tlf_ae_specific <- function(outdata,
                             text_font_size = 9,
                             orientation = "portrait",
                             footnotes = NULL,
+                            title = NULL,
```

Comment:
can I get an opinion on the title override? is this OK to leave in?

Reply:
yes, I feel it is a good design to allow user customize the title. 

We can provide a default value by using `metalite::collect_title`.  

https://merck.github.io/metalite/reference/collect_title.html

Reply:
So you'd suggest the function declaration having `title = metalite::collect_title(meta, ....)` as the argument default?

Reply:
Alternatively, you can write the part in the function body as below to avoid arguments depends on each other. 

```
if(is.null(title)){
   metalite::collect_title(meta, ....)
}
```